doc/build-windows.md

```diff
@@ -68,7 +68,7 @@ To build executables for Windows 64-bit, install the following dependencies:
 Then build using:
 
     cd depends
-    make HOST=x86_64-w64-mingw32
+    make HOST=x86_64-w64-mingw32 PATH=$(getconf PATH)
```

Comment:
Probably. I wasn't able to test the 32-bit version, so I didn't make the addition here, but I suspect that the command will be plagued by a similar issue.